frontend/src/lib/components/widgets/ChatInput/ChatInput.test.tsx

```diff
@@ -197,4 +197,53 @@ describe("ChatInput widget", () => {
     expect(textareas.length).toEqual(1)
     expect(textareas[0].value).toEqual("")
   })
+
+  it("disables the textarea and button", () => {
+    const props = getProps({ disabled: true })
+    const { container } = render(<ChatInput {...props} />)
+    const textareas = container.getElementsByTagName("textarea")
+    expect(textareas.length).toEqual(1)
+    expect(textareas[0].disabled).toBeTruthy()
```

Comment:
Nit: Would prefer the more semantic [@testing-library/jest-dom](https://www.npmjs.com/package/@testing-library/jest-dom#tobedisabled) matchers: `toBeDisabled()` & `toBeEnabled()`